### PR TITLE
Create BrianHCombes.json

### PR DESCRIPTION
**Pull request by BrianHCombes**
Adding what comes next

@githubteacher

closes #2887

### CODE REVIEW

**Comment by ZoeWard15891** [comment]
@BrianHCombes good job! :+1:

**Comment by GiulianoVitor** [comment]
Hello Brian, do you know why "checks" failed?

File: _pins/BrianHCombes.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+ githubHandle: BrianHCombes
+ latitude: 39.7055
+ longitude: -104.9664
+ ---
```
Comment (crichID) [inline _pins/BrianHCombes.json]: @BrianHCombes you will need to remove the leading space on this line to get your checks to pass! If you need help, I created a short video with the steps here: https://vimeo.com/201011288

**Comment by BrianHCombes** [comment]
Giuliano,
I don't know why the checks failed. However, I'm spanking new to all this. Got the concept down. But much fog has yet to lift. Only a matter of time for I need to master this.
I'm gonna take the lesson again.
Thanks!  Best Regards 

 Brian H. Combes
Full Stack Web Developer 
 Email:briancombes59@yahoo.com
 Phone:303-919-2307
 LinkedIn:My profile 

    On Wednesday, January 25, 2017 2:41 PM, Giuliano <notifications@github.com> wrote:
 

 Hello Brian, do you know why "checks" failed?—

**Comment by GiulianoVitor** [comment]
Hey Brian, In my case the file syntax was wrong (for example, I've used '.' instead of '-' and Capital letter for "Latitude" and "Longitude", the correct are "latitude: " and "longitude: "). I've made the fix, a new commit and the check passed and enabled the "merge button".


Best Regards from Brasil.

Giuliano

________________________________
De: Brian H Combes <notifications@github.com>
Enviado: quinta-feira, 26 de janeiro de 2017 04:35:20
Para: githubschool/open-enrollment-classes-introduction-to-github
Cc: Giuliano; Comment
Assunto: Re: [githubschool/open-enrollment-classes-introduction-to-github] Create BrianHCombes.json (#3158)

Giuliano,
I don't know why the checks failed. However, I'm spanking new to all this. Got the concept down. But much fog has yet to lift. Only a matter of time for I need to master this.
I'm gonna take the lesson again.
Thanks!  Best Regards

Brian H. Combes
Full Stack Web Developer
Email:briancombes59@yahoo.com
Phone:303-919-2307
LinkedIn:My profile

On Wednesday, January 25, 2017 2:41 PM, Giuliano <notifications@github.com> wrote:


Hello Brian, do you know why "checks" failed?-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


-

**Pull request merged**